src/app/drive/views/DriveView/DriveView.tsx

```diff
@@ -81,47 +80,14 @@ const DriveView = (props: DriveViewProps) => {
     }
   };
 
-  const breadcrumbItems = (): BreadcrumbItemData[] => {
-    const items: BreadcrumbItemData[] = [];
-
-    if (namePath.length > 0) {
-      const firstPath = namePath[0];
-
-      items.push({
-        id: firstPath.id,
-        label: t('sideNav.drive'),
-        icon: null,
-        active: true,
-        isFirstPath: true,
-        onClick: () => {
-          dispatch(uiActions.setIsGlobalSearch(false));
-          dispatch(storageThunks.goToFolderThunk(firstPath));
-          navigationService.push(AppView.Drive);
-        },
-      });
-
-      namePath.slice(1).forEach((path: FolderPath, i: number, namePath: FolderPath[]) => {
-        items.push({
-          id: path.id,
-          label: path.name,
-          icon: null,
-          active: i < namePath.length - 1,
-          onClick: () => navigationService.pushFolder(path.uuid),
-        });
-      });
-    }
-
-    return items;
-  };
-
   return (
     <>
       <Helmet>
         <title>{title}</title>
         <link rel="canonical" href={`${process.env.REACT_APP_HOSTNAME}`} />
       </Helmet>
       <DriveExplorer
-        title={<Breadcrumbs items={breadcrumbItems()} rootBreadcrumbItemDataCy="driveViewRootBreadcrumb" />}
+        title={<BreadcrumbsDriveView namePath={namePath} dispatch={dispatch} />}
```

Comment:
Could use directly the dispatch inside the BreacbrumsDriveView container instead of passing by props

Reply:
Done